src/UpdateVisuals.ts

```diff
@@ -542,7 +542,9 @@ export const visualUpdateOcteracts = () => {
         return
     }
     DOMCacheGetOrSet('singOcts').textContent = format(player.wowOcteracts, 2, true, true, true)
-    DOMCacheGetOrSet('sPO').textContent = format(1 / octeractGainPerSecond(), 2, true)
+    DOMCacheGetOrSet('sPO0').textContent = (octeractGainPerSecond() < 1) ? '+1 per' : 'You gain'
```

Comment:
+1 every

Reply:
Changed, guess that's what i get for following Plat's guidline